src/applications/personalization/profile/components/notification-settings/NotificationRadioButtons.jsx

```diff
@@ -189,12 +189,12 @@ const NotificationRadioButtons = ({
   const fieldsetClass = classNames(
     'rb-fieldset-input',
     'rb-input',
-    additionalFieldsetClass,
     {
       'rb-input-error': errorMessage,
       'rb-input-warning': warningMessage,
       'rb-input-success': successMessage,
     },
+    additionalFieldsetClass,

```

Comment:
this should be last so that it can potentially override any of the default styling